lint/deprecated_member_analyzer_test.go

```diff
@@ -30,18 +30,21 @@ import (
 
 func TestDeprecatedMemberAnalyzer(t *testing.T) {
 
+	// TODO: no more deprecated members
+	t.SkipNow()
```

Comment:
We would have no deprecated members in stable-cadence. So I guess we can remove this test.
However, the source-code for the analyzer can still be kept, since it might be needed in the future.